src/OpenIddict.EF/OpenIddictStore.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Identity.EntityFramework;
+using Microsoft.Data.Entity;
+using OpenIddict.Models;
+
+namespace OpenIddict {
+    public class OpenIddictStore<TUser, TApplication, TRole, TKey> : UserStore<TUser, TRole, OpenIddictContext<TUser, TApplication, TRole, TKey>, TKey>, IOpenIddictStore<TUser, TApplication>
```

Comment:
I don't like that we inherit from UserStore here. I would prefer to have 2 separate stores, the user and the OpenIddict one.